### ISSUE TITLE
Choosing league

### ISSUE THREAD

**Issue by ephankim** [bnorick/labbie#15]
Please add some choices in the setup to choose the league when pressing "price check"

**Comment by bnorick** [comment]
I can do that, but the helm data comes from poe.ninja for the current league only (they don't have data for standard). Does it still make sense to have a league selection for price checking?

**Comment by aiolos01** [comment]
Yes it absolutely does make sense. I current;ly play on standard and when I search it defaults to league and I need to change it to standard for every search. Sometimes this makes me hit the search engine rate limiting and have to wait.

I understand the data is from the league. There's nothing to be done about that, but at least you should be able to search for prices on standard.

P.S. Thanks for this great time saving utility.
